kolibri/auth/serializers.py

```diff
@@ -79,6 +79,11 @@ class ClassroomSerializer(serializers.ModelSerializer):
     coach_count = serializers.SerializerMethodField()
     admin_count = serializers.SerializerMethodField()
 
+    def validate_name(self, value):
```

Comment:
One alternative here would be to use the UniqueTogetherValidator on `name` and `parent`:
http://www.django-rest-framework.org/api-guide/validators/#uniquetogethervalidator